pages/index.js

```diff
@@ -39,12 +43,23 @@ export default function Home({
     closeOverlay();
   };
 
+  /////////////////tracking the number of checkedgifts
+  ////////////////update the number as soon as giftlist changes
+  const [numberCheckedGifts, setNumberCheckedGifts] = useState("");
+  useEffect(() => {
+    const numberCheckedGifts = giftList.filter((gift) => {
+      return gift.isChecked === true;
+    });
+    setNumberCheckedGifts(numberCheckedGifts.length);
+  }, [giftList]);
+
   return (
     <>
       <Header headertitle="Home" />
       <MenuPositioning>
         <H2andButtonFlex>
-          <H2>Gift ideas in your list: {giftList.length}</H2>
+          <H2>Gift ideas: {giftList.length}</H2>
+          <button disabled={numberCheckedGifts <= 0}>New vote</button>
```

Comment:
I see. I would recommend not doing here "logic". I would doing it where you count the number

```js
// from earlier ;-)
const checkedGifts = giftList.filter((gift) => gift.isChecked === true );
// I would change 
const numberCheckedGifts = checkedGifts.length
// to this
const isDisabled = checkedGifts.length <= 0
```
and then
```suggestion
          <button disabled={isDisabled}>New vote</button>
```

Reply:
done